Domain/Music_Portal.Domain.Core/ApplicationContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace Domain.Core
```

Comment:
The namespace should start with the Project name and then internal folders and e.c. like
`namespace Music_Portal.Domain.Core`